presto-spi/src/main/java/com/facebook/presto/spi/security/Privilege.java

```diff
@@ -15,5 +15,5 @@
 
 public enum Privilege
 {
-    SELECT, DELETE, INSERT
+    SELECT, DELETE, INSERT, UPDATE
```

Comment:
If I don't add it here, `SHOW GRANTS` won't show the privilege, even if user possesses it, for ex., in the case of the owner of a Hive table who has all the privileges, including the `UPDATE` privilege. I can add a product test, testing whether `UPDATE` is shown for the owner of the table.